Update @font-face page (#21570)
mdn/content: files/en-us/web/css/@font-face/index.md

```css
@font-face {
  font-family: "Trickster";
  src: local("Trickster"),
       url("trickster-COLRv1.otf") format("opentype") tech(color-COLRv1),

>        url("trickster-outline.otf") format("opentype");

Reviewer: ```suggestion
       url("trickster-outline.otf") format("opentype"),
       url("trickster-outline.woff") format("woff");
```

- {{cssxref("@font-face/size-adjust", "size-adjust")}}
  - : Defines a multiplier for glyph outlines and metrics associated with this font. This makes it easier to harmonize the designs of various fonts when rendered at the same font size.
- {{cssxref("@font-face/src", "src")}}

>   - : Specifies font resources. Is a comma-separated list representing the resource fallback order, each component containing a `url()` or a `local()`. `url()` can be followed by `format()` and `tech()`, like this:

Reviewer: ```suggestion
  - : Specifies font resources. A comma-separated list representing the resource fallback order, each font file within  a `url()` or `local()`.  The `url()` can be followed by `format()` and `tech()`, like this:
```

    ```css
    src: local("Trickster"),
         url("trickster-COLRv1.otf") format("opentype") tech(color-COLRv1),

>          url("trickster-outline.otf") format("opentype");

Reviewer: ```suggestion
         url("trickster-outline.otf") format("opentype"),
         url("trickster-outline.woff") format("woff");
```

    ```

>     `local()`: Specifies the font name to not download the font when a local font is available. The font name may or may not be enclosed in quotes.

Reviewer: ```suggestion
    `local()`: Specifies the font name for when the font is available in the user's operating system. Quoting the font name is optional.
```

>     `format()`: **Optional**. Specify the font format. If the value is invalid, the browser may not download the resource; but if this parameter is not provided, the browser will always download the resource and then detect the format. The value can be a _keyword_ or a _string_, _keyword_ mean no quotes, _string_ require quotes. _keyword_ is newer specification, so perhaps not yet widely supported. _keyword_ support: `collection`, `embedded-opentype`, `opentype`, `svg`, `truetype`, `woff`, `woff2`. See below for supported _string_.

Reviewer: ```suggestion
    `format()`: **Optional**. Specifies the font format. If the value is not supported or invalid, the browser may not download the resource, potentially saving bandwidth. If omitted, the browser will always download the resource and then detect the format. The value can be a _keyword_ or a _string_, (_strings_ require quotes, _keywords_ do not). Keywords support include  `collection`, `embedded-opentype`, `opentype`, `svg`, `truetype`, `woff`, `woff2`. See the table below for supported _string_ values.
```

    `tech()`: **Optional**. Not yet widely supported. Value is a _keyword_, support: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-COLRv0`, `color-COLRv1`, `color-SVG`, `color-sbix`, `color-CBDT`.

    The following are _string_ values supported by `format()`, and their new syntax equivalents:

    | String form                     | Equivalent syntax                   |
    | ------------------------------- | ----------------------------------- |

>     | `format("woff2")`               | `format(woff2)`                     |

Reviewer: ```suggestion
    | `format("svg")`               | `format(svg)`                     |
    | `format("woff2")`               | `format(woff2)`                     |
```

    | `format("woff2-variations")`    | `format(woff2) tech(variations)`    |
    | `format("woff-variations")`     | `format(woff) tech(variations)`     |
    | `format("truetype-variations")` | `format(truetype) tech(variations)` |
    | `format("opentype-variations")` | `format(opentype) tech(variations)` |

- {{cssxref("@font-face/unicode-range", "unicode-range")}}
  - : The range of Unicode code points to be used from the font.

## Description

If the `local()` function is provided, specifying a font name to look for on the user's computer, and if the {{Glossary("user agent")}} finds a match, that local font is used. Otherwise, the font resource specified using the `url()` function is downloaded and used.

By allowing authors to provide their own fonts, `@font-face` makes it possible to design content without being limited to the so-called "web-safe" fonts (that is, the fonts which are so common that they're considered to be universally available). The ability to specify the name of a locally-installed font to look for and use makes it possible to customize the font beyond the basics while making it possible to do so without relying on an Internet connection.

> It's common to use both `url()` and `local()` together, so that the user's installed copy of the font is used if available, falling back to downloading a copy of the font if it's not found on the user's device.

Reviewer: same comment as above.

- {{cssxref("@font-face/size-adjust", "size-adjust")}}
  - : Defines a multiplier for glyph outlines and metrics associated with this font. This makes it easier to harmonize the designs of various fonts when rendered at the same font size.
- {{cssxref("@font-face/src", "src")}}
  - : Specifies font resources. A comma-separated list representing the resource fallback order, each resource specified by a `url()` or `local()`. If the previous resource is loaded successfully, the latter resources will not be used. The `url()` can be followed by `format()` and `tech()`, like this:

    ```css
    src: local("Trickster"),
         url("trickster-COLRv1.otf") format("opentype") tech(color-COLRv1),
         url("trickster-outline.otf") format("opentype"),
         url("trickster-outline.woff") format("woff");
    ```

>     `url()`: Specifies the URL of a font file, like any other `url()` in CSS. If the font file used as a container contains multiple fonts, one of them should be specified, as follows:

Reviewer: ```suggestion
    `url()`: Specifies the URL of a font file, like any other `url()` in CSS. If the font file is a container for multiple fonts, a fragment identifier is included to indicate which sub-font should be used, as follows:
```

    ```css
    src: url(collection.otc#WhichFont); /* WhichFont is the PostScript name of a font in the font file */
    src: url(fonts.svg#WhichFont);      /* WhichFont is the element id of a font in the SVG Font file */
    ```

>     `local()`: Specifies a font name for when the font is available in the user's device. Quoting the font name is optional.

Reviewer: ```suggestion
    `local()`: Specifies the font name should the font be available on the user's device. Quoting the font name is optional.
```
